### ISSUE TITLE
Backticks in imports should be ignored for ordering purposes

### ISSUE THREAD

**Issue by bethcutler** [pinterest/ktlint#1106]
## Expected Behavior
Imports are ordered alphabetically regardless of quotes

```kotlin
import org.mockito.Mockito.verify
import org.mockito.Mockito.`when`
```

## Observed Behavior
Imports with backticks are sorted above any other imports at the same level

```kotlin
import org.mockito.Mockito.`when`
import org.mockito.Mockito.verify
```

## Steps to Reproduce
(See code samples above)

## Your Environment
* Version of ktlint used: 0.40.0
* Name and version (or code for custom task) of integration used (Gradle plugin, Maven plugin, command line, custom Gradle task): command line

**Comment by paulferaud** [comment]
Probably rarer, but it would also be possible for the escapes to appear as part of the package to.
```
import org.library.`typealias`.someImport
```

**Comment by ganadist** [comment]
Kotlin Plugin for IntelliJ still provides old imports ordering behavior.
https://youtrack.jetbrains.com/issue/KTIJ-20283

**Comment by nbadal** [comment]
I have submitted a change to the JetBrains Kotlin plugin to match this ordering, and it will eventually be released in an update 😁

**Comment by rocketraman** [comment]
> I have submitted a change to the JetBrains Kotlin plugin to match this ordering, and it will eventually be released in an update grin

For reference:

https://youtrack.jetbrains.com/issue/KTIJ-19394
https://github.com/JetBrains/intellij-community/pull/1879

According to YouTrack, should be part of 2022.1.

**Comment by hrach** [comment]
Do I understand correctly that you have created inconsistency between IntelliJ platform and ktlint that will last at least a year before Android Studio gets stable on 2022.1 platform? And without any possibility to configure this? 

Is there a chance that some AS will get it sooner? Alternatively, would you be open to accept PR allowing to revert this with some configuration?

**Comment by bogdanzurac** [comment]
Can confirm that we've hit this issue as well on Android as @hrach pointed out.

It's indeed a bit weird that this change was initially done in June 2021, with support inside IDEA not even out yet (coming in roughly a full year after the initial change in KtLint) and with AS getting the update based off of the new IDEA version probably after 1 more year, in 2023. So almost 2 years difference in time between the 2 dependencies (KtLint and actual IDE)...

Are there any contingencies in place for de-syncing issues like this? Can this be mitigated somehow for both the current situation as well as any possible future cases?

**Comment by adamdanielczyk** [comment]
Agreed with @hrach & @bogdanzurac  - since the intellij plugin fix is in IDEA release 2022.1, which is only included in Android Studio Electric Eel that's still in canary channel, it will still take a lot of time before that hits stable channel.

Please provide a way to disable this inconsistency or revert the change as proposed above. Unless there's an other known way to solve this?
